internal/dinosaur/pkg/clusters/standalone_provider.go

```diff
@@ -463,14 +464,14 @@ func applyResource(dynamicClient dynamic.Interface, mapper *restmapper.DeferredD
 	// parse resource obj to unstructure.Unstructered
 	data, err := json.Marshal(resource)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("marshalling resource: %w", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("marshaling resource %q into JSON: %w", resource.???, err)
```

Is it easy to get a kind & name combination or something like that for a generic `resource interface{}`?

Reply:
Actually it is, something like ("%T %v", resource, resource) would print the type and value. Should I do this?